app/src/main/java/co/condorlabs/customcomponents/Constants.kt

```diff
@@ -193,7 +193,7 @@ const val TITLE_TYPE = 1
 const val SUBTITLE_TYPE= 2
 const val SECTION_TITLE_TYPE = 3
 const val LINK_TYPE = 4
-
+const val MILLISECONDS_TO_SHOW_PLACE_HOLDER = 200.toLong()
```

Comment:
**Suggestion**: `200L`

Reply:
is the same whats the difference or the benefit to use it in this way ?